### DISCUSSION TITLE
Does ente photos support full offline use locally without a server or docker image?

### DISCUSSION THREAD

**Discussion by Fuoxden** [ente-io/ente#3083] [Q&A] [answered]
I don't want to sync backups to the cloud, I don't want nor care for Automatic backups, Can I use ente photos locally? On my device without a server or even a docker image, can I use ente fully on my device without network connection?

**Comment by mnvr** [comment, marked as answer]
If you're not self-hosting, then no, not currently.

**Reply by Fuoxden** [reply]
Okay, but I do want to ask, do you ever have plans to ever add a local mode? I just want you to clarify, I do think it's weird that ente photos doesn't have a local mode, I believe stuff like apples photos app has local support, and other photos apps have local mode, I'm just wondering cuz basically no one has ever asked

**Comment by candideu** [comment]
I'm also curious about this!
